maintain/going-production-with-dcos.md

```diff
@@ -273,6 +273,7 @@ TODO: *Update the above page to show runc*
 
 * Work with Portworx Support before planning major upgrades. Ensure all volumes have the latest snapshots before performing upgrade
 * Ensure there are [cloudsnaps](https://docs.portworx.com/cloud/backups.html) that are taken.
+* Due to a [bug in DCOS](https://jira.mesosphere.com/browse/DCOS_OSS-2103) the Portworx service file will get removed after an upgrade. Please run `dcos portworx plan force-restart deploy portworx-deploy` after the DCOS upgrade to re-deploy the service file.
```

Comment:
We will have to use portworx-install once the new framework is released

Reply:
Yes, this won't be needed with the updates to the framework.